docs/book/src/cronjob-tutorial/cronjob-tutorial.md

```diff
@@ -46,7 +46,7 @@ project:
 ```bash
 # we'll use a domain of tutorial.kubebuilder.io,
 # so all API groups will be <group>.tutorial.kubebuilder.io.
-kubebuilder init --domain tutorial.kubebuilder.io
+kubebuilder init --plugins=go/v3-alpha --domain tutorial.kubebuilder.io
```

Comment:
Assuming we'll update this post `stability` time?

Reply:
Yes. My idea is to update that for go/v3 when its stable version is released.